CONTRIBUTING.md

```diff
@@ -0,0 +1,24 @@
+# Contributing to Secure Quick Reliable Login.
+
+First of thank you for considering helping out with this project. All contributions are welcome, we want this project to 
+reach completion with a full implementation of the SQRL specification.
+
+### I want to file a bug.
+
+Then you open an issue under [issues](https://github.com/kalaspuffar/secure-quick-reliable-login/issues) and follow the 
+guide there. Please search before you post so we get as few duplicates as possible.
+
+### I want to suggest a feature.
+
+Then you open an issue under [issues](https://github.com/kalaspuffar/secure-quick-reliable-login/issues) and follow the 
+guide there. Please search before you post so we get as few duplicates as possible.
+
+### Code style
+
+We are trying to keep an consistant coding convention in order to help readablility and make it easy to submit code.
+
+Things to refrain from:
```

Comment:
The coding style should express the desired behaviour. Instead of listing things you must not do, you should be clear and mention what you should do.
Please rewrite so that this line starts with "Try to stick to:"